config/routes.php

```diff
@@ -48,6 +48,17 @@
     ],
 
 
+    '/api/branchOffice/' => [
+        'GET' => [BranchOffice::class, 'apiData']
```

Comment:
Usually it's more common to have separated controller instead of separated method

Reply:
New class for API has been created